tests/cisco_ios/show_lldp_neighbors_detail/cisco_ios_show_lldp_neighbors_detail6.yml

```diff
@@ -0,0 +1,23 @@
+---
+CAPABILITIES: T
```

Comment:
ntc_templates will convert these to lowercase, so these will need to be updated to be lowercase.